recipes/perl-params-coerce/0.14/meta.yaml

```diff
@@ -0,0 +1,33 @@
+{% set name = "perl-params-coerce" %}
+{% set version = "0.14" %}
+{% set sha256 = "5c67b11b9859e8d0d048c1c529e16ff49458fe4a98a16ccbfec6cd66e49b7713" %}
+
+package:
+  name: {{ name }}
+  version: {{ version }}
+
+source:
+  fn: Params-Coerce-0.14.tar.gz
+  url: https://cpan.metacpan.org/authors/id/A/AD/ADAMK/Params-Coerce-0.14.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+
+requirements:
+  build:
+    - perl #>=5.22.1
+    - perl-params-util
+
+  run:
+    - perl #>=5.22.1
```

Comment:
is this a comment? You can pin your recipe if you want to restrict it to specific perl versions. Would it make sense to also include a <6.0?

Reply:
this was just for testing. Restriction <6.0 definitely makes sense.

Reply:
You can for example do something like this: `>=5.22,<6.0`

Reply:
in this case, is it necessary to increase the build number from 0 to 1?

Reply:
No, the build number only needs to be increased if there is already a package with this version and you want o have a new-fixed one.